dom/attr.js

```diff
@@ -170,6 +172,27 @@ export function getData (element, key)
 }
 
 
+/**
+ * Determines whether the given data attribute is set on the element
+ *
+ * @param {HTMLElement} element
+ * @param {string} key
+ * @returns {boolean}
+ */
+export function hasData (element, key)
+{
+    const value = getData(element, key);
+
+    // Empty data attributes' value is an empty string
+    if ("" === value)
+    {
+        return true;
+    }
+
+    return null !== (value || null);
```

Comment:
`return null != value`

Reply:
Ah, that one was still in there due to the previous implementation, which could return `undefined`. Fixed :)

Reply:
The short check `!=` is important, as it catches both `null` and `undefined`.